samples/vsphere/sso/embedded_psc_sso_workflow.py

```diff
@@ -100,14 +105,44 @@ def run(self):
         session_sec_ctx = create_session_security_context(self.session_id)
         connector.set_security_context(session_sec_ctx)
 
-        print('\nStep 3: List available datacenters using the vAPI services')
+        # Create and Delete Tag to Verify connection is successful
+        print('\nStep 3: Creating and Deleting Tag Category and Tag...\n')
+        self.tag_svc = Tag(stub_config)
+        self.category_svc = Category(stub_config)
+
+        self.category_id = self.create_tag_category('TestTagCat', 'TestTagDesc',
+                                                    CategoryModel.Cardinality.MULTIPLE)
+        assert self.category_id is not None
+        print('Tag category created; Id: {0}'.format(self.category_id))
 
-        datacenter_svc = Datacenter(stub_config)
-        pprint(datacenter_svc.list())
+        self.tag_id = self.create_tag('TestTag', 'TestTagDesc', self.category_id)
```

Comment:
We just need to make a most basic API call to show we login successfully. So I think we can skip the tag creation step and just create a category and then delete it. 

I think it's ok to have the create category method here. No need to create a helper method in common package as we are not using it that often.